chromaprint.py

```diff
@@ -25,7 +25,7 @@ def _guess_lib_name():
     if sys.platform == 'darwin':
         return ('libchromaprint.1.dylib', 'libchromaprint.0.dylib')
     elif sys.platform == 'win32':
-        return ('chromaprint.dll', 'libchromaprint.dll')
+        return ('', 'libchromaprint.dll')
```

Comment:
I'd like to understand more about the reason to delete this name. It sounds like, in your system's setup, only the latter name is used. But was there a specific problem with including the other name (uselessly, in your case)? Did it slow things down or trigger an error?

Reply:
In `C:\Program Files (x86)\chromaprint\bin` I only have `libchromaprint.dll`, but that gives no problems to execution.

Reply:
OK! Maybe we should just leave this first library name there then, if it's not causing any problems?

Reply:
Yes, ok, in the first PR I removed it while i was indagating.
Maybe we could switch the library positions in order to make any troubleshooting not to be deviated by `chromaprint.dll` nonexistence.

Reply:
I'm conceptually OK with swapping the order, but I still don't quite understand what problem it solves. Are there error messages or anything that appear when trying to load a non-existent library? Because if so, it seems like we should work on suppressing those…

Reply:
No error nor warning at all, it was a mistake on my side thinking it could change something.
Tell me what do you want to do with this.

Reply:
OK, cool! Let's just revert this and leave it as-is until it becomes a problem.

Reply:
OK